$NetBSD: patch-apps_tqsl_cpp,v 1.1 2013/09/13 08:45:52 mef Exp $

(1)
Avoid following error (interim, only valid with db5)
/usr/pkgsrc/wip/trustedQSL/work/tqsl-2.0/apps/tqsl.cpp:1133:36: error: 'DB_VERSION_STRING' was not declared in this scope
gmake[2]: *** [apps/CMakeFiles/tqsl.dir/tqsl.cpp.o] Error 1
gmake[1]: *** [apps/CMakeFiles/tqsl.dir/all] Error 2

(2) Couldn't upload the file: CURL returned 
"Peer certificate cannot be authenticated with given CA certificates" 
(SSL certificate problem: self signed certificate in certificate chain)

--- apps/tqsl.cpp.orig	2013-10-21 09:33:20.000000000 +0900
+++ apps/tqsl.cpp	2013-11-29 00:05:45.000000000 +0900
@@ -71,7 +71,7 @@
 #endif
 #include <zlib.h>
 #include <openssl/opensslv.h> // only for version info!
-#include <db.h> //only for version info!
+#include <db5/db.h> //only for version info!
 #include "tqslwiz.h"
 #include "qsodatadialog.h"
 #include "tqslerrno.h"
@@ -2084,6 +2084,9 @@ tqsl_curl_init(const char *logTitle, con
 	//set up options
 	curl_easy_setopt(req, CURLOPT_URL, url);
 
+#if 1  //def NO_VERYFYPEER	  
+	curl_easy_setopt(req, CURLOPT_SSL_VERIFYPEER, FALSE); 
+#endif
 	// Get the proxy configuration and pass it to cURL
         wxConfig *config = (wxConfig *)wxConfig::Get();
         config->SetPath(wxT("/Proxy"));
@@ -2154,8 +2157,10 @@ retry_upload:
 
 	curl_easy_setopt(req, CURLOPT_WRITEFUNCTION, &FileUploadHandler::recv);
 	curl_easy_setopt(req, CURLOPT_WRITEDATA, &handler);
+#if 0
 	curl_easy_setopt(req, CURLOPT_SSL_VERIFYPEER, uplVerifyCA);
-
+#endif
+ 
 	char errorbuf[CURL_ERROR_SIZE];
 	curl_easy_setopt(req, CURLOPT_ERRORBUFFER, errorbuf);
 
